content/kubermatic/main/tutorials-howtos/networking/cni-cluster-network/_index.en.md

```diff
@@ -51,35 +47,89 @@ In KKP versions below v2.19, this was the only supported CNI.
 
 **NOTE:** IPVS kube-proxy mode is not recommended with Cilium CNI due to [a known issue]({{< relref "../../../architecture/known-issues/" >}}#2-connectivity-issue-in-pod-to-nodeport-service-in-cilium--ipvs-proxy-mode).
 
-To provide better observability on cluster networking with Cilium CNI via a web user interface, KKP provides a Hubble Addon that can be easily installed into user clusters with Cilium CNI via the KKP UI on the cluster page, as shown below:
-
-![Cluster Details - Addons](/img/kubermatic/main/tutorials/networking/ui_addons.png?classes=shadow,border "Cluster Details - Addons")
-
-![Cluster Details - Addons - Install Addon](/img/kubermatic/main/tutorials/networking/ui_addon_hubble.png?classes=shadow,border "Cluster Details - Addons - Install Addon")
-
-After the Hubble addon is installed into the cluster, the Hubble UI can be displayed by port-forwarding to it, e.g.:
+To allow better observability and troubleshooting of cluster networking with Cilium CNI, Cilium is by default deployed with the [Hubble user interface](https://github.com/cilium/hubble-ui). To access Hubble UI, you can use port-forwarding, e.g.:
 
 ```bash
 kubectl port-forward -n kube-system svc/hubble-ui 12000:80
 ```
 
 After the above port-forwarding is active, the Hubble UI can be shown by navigating to the URL [http://localhost:12000](http://localhost:12000).
 
-Please note that to have the Hubble addon available, the KKP installation has to be configured with `hubble` as [an accessible addon]({{< relref "../../../architecture/concept/kkp-concepts/addons/#accessible-addons" >}}).
+Please note that for Cilium versions below `1.13.0`, Hubble had to be installed as a KKP Addon. As of Cilium `1.13.0` it is enabled by default, but can be disabled if necessary. See [Deploying CNI as a System Application](#deploying-cni-as-a-system-application) for more details.
 
 ### None CNI
 
 "None" CNI is a special KKP-internal CNI type, which does not install any CNI managed by KKP into the user cluster. CNI management is therefore left on the cluster admin which provides a flexible option to install any CNI with any specific configuration.
 
-When this option is selected, the user cluster will be left without any CNI, and will not be functioning until some CNI is installed into it by the cluster admin. This can be done either manually (e.g. via helm charts), or by leveraging KKP [Addons]({{< relref "../../../architecture/concept/kkp-concepts/addons/#accessible-addons" >}}) infrastructure and creating a custom Accessible addon.
+When this option is selected, the user cluster will be left without any CNI, and will not be functioning until some CNI is installed into it by the cluster admin. This can be done either manually (e.g. via helm charts), or by leveraging the KKP [Accessible Addons]({{< relref "../../../architecture/concept/kkp-concepts/addons/#accessible-addons" >}}) infrastructure or the [Applications]({{< relref "../../applications" >}}) feature.
```

Comment:
```suggestion
When this option is selected, the user cluster will be left without any CNI, and will not be functioning until some CNI is installed into it by the cluster admin. This can be done either manually (e.g. via helm charts), or by leveraging the KKP [Accessible Addons]({{< relref "../../../architecture/concept/kkp-concepts/addons/#accessible-addons" >}}) infrastructure or the [Applications]({{< relref "../../../architecture/concept/kkp-concepts/applications" >}}) feature.
```

Applications documentation has been split between concepts and tutorials. You may have to rebase your PR to get the concept part.